.github/dependabot.yml

```diff
@@ -8,6 +8,8 @@ updates:
     open-pull-requests-limit: 20
     reviewers:
       - "mongodb/mongocli"
+    ignore:
+      - dependency-name: "go.mongodb.org/atlas-sdk/*"
```

Comment:
ignoring on dependabot

Reply:
Actualy I'm not sure that we should do this. Do we have other job covering minor updates?

Reply:
this one